examples/typegraphs/http-runtime.py

```diff
@@ -0,0 +1,44 @@
+# skip:start
```

Comment:
Do we need to skip the imports? shouldn't we keep them as a reference incase the user imports from the wrong file/lib?

Reply:
This was a not optimal solutions, ideally, we should have the import small so the core the typegraph is the focus point, yet having a fully complete code